tests/simulator/advance/test_advance_mhd.py

```
@@ -0,0 +1,151 @@
+
+from tests.diagnostic import all_timestamps
+from tests.simulator.test_advance import AdvanceTestBase
+
+
+class MHDAdvanceTest(AdvanceTestBase):
+    def getHierarchy(
+        self,
+        ndim,
+        interp_order,  # torm?
+        qty,
+        refinement_boxes={},
```

Ruff (0.15.11)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)